docs/docs/error-codes.md

```diff
@@ -0,0 +1,15 @@
+---
+id: error-codes
+title: React Error Codes
+permalink: error-codes.html
+---
+
+If you're here because you found an error link in your console, read on!
```

Comment:
Instead of these two paragraphs, can we do:

> In the minified production build of React, we avoid sending down full error messages in order to reduce the number of bytes sent over the wire. We highly recommend using the development build locally when debugging your app since it tracks additional debug info and provides helpful warnings about potential problems in your apps, but if you encounter an exception while using the production build, this page will reassemble the original text of the error.